### ISSUE TITLE
Remove case sensitivity from api calls

### ISSUE THREAD

**Issue by jon64digital** [storyblok/storyblok#513]
**The feature would affect:** (check one with "x")   
- [ ] *app.storyblok.com (CMS - Interface)*   
- [ ] *api.storyblok.com (CMS - Content Delivery API)*
- [ ] *mapi.storyblok.com (CMS - Management API)*
- [ ] *capi.storyblok.com (Commerce - API)*     
- [ ] *Commerce - Interface*   
- [ ] *Other* 

**Is your feature request related to a problem? Please describe.**

Calls to content delivery URLs are case sensitive, which means that if a user types a capital letter into the URL of one of my pages and the slug is used to construct the Storyblok url i have to make sure I always .toLowerCase() it, otherwise errors occur. lLugs are URLs and URLs should not be case-sensitive.

**Describe the solution you'd like**

Basically, /products/widgets/blue-widget should be resolved if the user  goes to domain.com/products/widgets/Blue-Widget

**Comment by alvarosabu** [comment]
Hi @JonCognioDigital, we are closing this issue because this repository was used in the early days for support-related questions, but we've since transitioned to more effective channels for helping our users.

Please, refer to the following resources:

Storyblok Github Org: https://github.com/storyblok
Storyblok Docs: https://www.storyblok.com/docs
Official Support: https://support.storyblok.com
Join our Discord: https://www.storyblok.com/join-discord

Thanks for the understanding 🙏🏻
